engine/lib/comments.php

```diff
@@ -431,6 +435,48 @@ function _elgg_comments_prepare_notification($hook, $type, $returnvalue, $params
 	return $returnvalue;
 }
 
+/**
+ * Adds comment menu items to entity menu
+ *
```

Comment:
Could you please make a habit of documenting the hook name. It's impossible to read the code without an IDE

Reply:
good point